views.py

```diff
@@ -1433,11 +1434,17 @@ def article_file_make_galley(request, article_id, file_id):
     """
     article_object = get_object_or_404(
         submission_models.Article, pk=article_id)
-    file_object = get_object_or_404(
+    janeway_file = get_object_or_404(
         core_models.File, pk=file_id)
 
-    journal_logic.create_galley_from_file(
```

Comment:
It seems that `create_galley_from_file` is no longer suitable for this operation. Should we add a note to the docstring saying it is deprecated?